cmd/kubeadm/app/util/apiclient/wait.go

```diff
@@ -246,7 +245,7 @@ func getStaticPodSingleHash(client clientset.Interface, nodeName string, compone
 		return "", err
 	}
 
-	staticPodHash := staticPod.Annotations[kubetypes.ConfigHashAnnotationKey]
+	staticPodHash := staticPod.Annotations["kubernetes.io/config.hash"]
```

Comment:
Thanks @liggitt !
Will make this a const once master is reopened for the 1.18 cycle.
/lgtm